src/main/java/com/vinyl/service/Validator.java

```diff
@@ -14,6 +14,10 @@
 	public static final String VALID_NAME_REGEX = "^[A-Z][a-zA-Z]+$";
 	private static final Pattern NAME_PATTERN = Pattern.compile(VALID_NAME_REGEX);
 
+	public static final String VALID_EMAIL_REGEX = "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@"
```

Comment:
encapsulation, if the fields are not used outside of this class, make them private

Reply:
done